openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTableV1.component.tsx

```diff
@@ -0,0 +1,925 @@
+        );
+      }
+    });
+  };
+
+  const handleEditColumnChange = (columnDescription: string): void => {
+    if (editColumn) {
+      const tableCols = cloneDeep(tableColumns);
+      updateColumnDescription(
+        tableCols,
+        editColumn.column.name,
+        columnDescription
+      );
+      onUpdate?.(tableCols);
+      setEditColumn(undefined);
+    } else {
+      setEditColumn(undefined);
+    }
+  };
+
+  const handleTagSelection = (
+    selectedTags?: Array<EntityTags>,
+    columnName = ''
+  ) => {
+    const newSelectedTags: TagOption[] | undefined = selectedTags?.map(
+      (tag) => {
+        return { fqn: tag.tagFQN, source: tag.source };
+      }
+    );
+    if (newSelectedTags && (editColumnTag || columnName)) {
+      const tableCols = cloneDeep(tableColumns);
+      updateColumnTags(
+        tableCols,
+        editColumnTag?.column.name || columnName,
+        newSelectedTags
+      );
+      onUpdate?.(tableCols);
+    }
+    setEditColumnTag(undefined);
+  };
+
+  const getFrequentlyJoinedWithColumns = (
+    columnName: string
+  ): Array<JoinedWith> => {
+    return (
+      joins.find((join) => join.columnName === columnName)?.joinedWith || []
+    );
+  };
+  const checkIfJoinsAvailable = (columnName: string): boolean => {
+    return (
+      joins &&
+      Boolean(joins.length) &&
+      Boolean(joins.find((join) => join.columnName === columnName))
+    );
+  };
+
+  const searchInColumns = (table: Column[], searchText: string): Column[] => {
+    const searchedValue: Column[] = table.reduce((searchedCols, column) => {
+      const isContainData =
+        lowerCase(column.name).includes(searchText) ||
+        lowerCase(column.description).includes(searchText) ||
+        lowerCase(getDataTypeString(column.dataType)).includes(searchText);
+
+      if (isContainData) {
+        return [...searchedCols, column];
+      } else if (!isUndefined(column.children)) {
+        const searchedChildren = searchInColumns(column.children, searchText);
+        if (searchedChildren.length > 0) {
+          return [
+            ...searchedCols,
+            {
+              ...column,
+              children: searchedChildren,
+            },
+          ];
+        }
+      }
+
+      return searchedCols;
+    }, [] as Column[]);
+
+    return searchedValue;
+  };
+
+  const checkPermission = () =>
+    isAdminUser ||
+    hasEditAccess ||
+    isAuthDisabled ||
+    hasPemission(Operation.EditDescription, EntityType.TABLE, userPermissions);
+  /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+  const getColumnName = (cell: any) => {
+    const fqn = cell?.fullyQualifiedName || '';
+    const columnName = getPartialNameFromTableFQN(fqn, [FqnPart.NestedColumn]);
+    // wrap it in quotes if dot is present
+
+    return columnName.includes(FQN_SEPARATOR_CHAR)
+      ? `"${columnName}"`
+      : columnName;
+  };
+
+  /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
```

Comment:
We are using `antd` table, So, Can we use `antd` cell type here and in other places as well?

Reply:
Changes made as suggested.